capa/features/extractors/cape/thread.py

```diff
@@ -24,5 +25,22 @@ def get_calls(ph: ProcessHandle, th: ThreadHandle) -> Iterator[CallHandle]:
         if call.thread_id != tid:
             continue
 
-        addr = DynamicCallAddress(thread=th.address, id=call_index)
-        yield CallHandle(address=addr, inner=call)
+        for symbol in generate_symbols(call.api):
+            call.api = symbol
+
+            addr = DynamicCallAddress(thread=th.address, id=call_index)
+            yield CallHandle(address=addr, inner=call)
```

Comment:
I've noticed we run into issues since CAPE reports only list the api (like `CreateFileA`), however:
* rules often include the DLL (which we won't get easily here)
* use the generic name (like `CreateFile` [not A/W])

This is one way/part of handling this, maybe we can come up with a more generic way.